sdk/core/azure-core/test/performance/inc/azure/core/test/performance/nullable.hpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ * @brief Test the Nullable component performance.
+ *
+ */
+
+#pragma once
+
+#include <azure/performance_framework.hpp>
+
+#include <memory>
+
+namespace Azure { namespace Core { namespace Test { namespace Performance {
+
+  /**
+   * @brief Measure the Nullable object performance.
+   *
+   */
+  class NullableTest : public Azure::PerformanceStress::PerformanceTest {
```

Comment:
I favor "Perf" over PerformanceStress/Performance.  Perf is very common name for Perf testing.  Its short and concise.  If you want to go with Performance that would be my second choice

Reply:
Rename the perf name: https://github.com/Azure/azure-sdk-for-cpp/issues/1696